package.json

```diff
@@ -5,12 +5,13 @@
   "license": "MIT",
   "epic-stack": true,
   "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com/)",
+	"type": "module",
   "scripts": {
     "build": "run-s build:*",
     "build:remix": "remix build",
     "build:server": "tsx ./other/build-server.ts",
     "dev": "remix dev -c \"npm run dev:server\" --no-restart",
-    "dev:server": "cross-env MOCKS=true tsx watch --clear-screen=false --ignore \"app/**\" --ignore \"build/**\" --ignore \"node_modules/**\" ./index.js",
+    "dev:server": "cross-env MOCKS=true tsx watch --clear-screen=false --ignore \"app/**\" --ignore \"build/**\" --ignore \"node_modules/**\" ./index.cjs",
```

Comment:
Any reason we can't make the entry file an ESM file with dynamic imports for the dynamic stuff?

Reply:
done